Index: src/app/index.js
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
--- src/app/index.js	(revision 30ff7b07231131e6d7d8174ff807dde17b7efffe)
+++ src/app/index.js	(date 1530282774672)
@@ -9,6 +9,7 @@
 import { App, store as cmfstore, actions } from '@talend/react-cmf';
 import { browserHistory } from 'react-router';
 import { routerMiddleware, syncHistoryWithStore } from 'react-router-redux';
+import logger from 'redux-logger';
 
 import configure from './configure';
 import appReducer from './reducers';
@@ -28,7 +29,7 @@
 /**
  * Register your app reducers
  */
-const store = cmfstore.initialize(appReducer);
+const store = cmfstore.initialize(appReducer, undefined, undefined, logger);
 
 /**
  * Fetch the CMF settings and configure the CMF app
